modules/nlu/src/backend/api.ts

```diff
@@ -196,6 +196,11 @@ export default async (bp: typeof sdk, state: NLUState) => {
     const { botId } = req.params
     const ghost = bp.ghost.forBot(botId)
     const entities = await getEntities(ghost)
+
+    if (req.query.asOption) {
```

Comment:
you ask the backend to format its json so it can be displayed in a dropdown ? that's quite weird

Reply:
Yes, because you can configure a condition to query an endpoint to get the elements to display in a dropdown. Since the frontend fetch the list of conditions from the backend, you can't pass a conversion method between them, so I figured that the best solution is to let the backend send the data in the correct format for the method to use. This way you only need to configure the endpoint and nothing else

Reply:
I understand but you'll agree it's weird that you have to tweak a backend route to display something. why not render a custom ui for this. I know you want to avoid this but we will not start change backend simply to render a dropdown ..

Reply:
Most of the time you won't have to change the backend route, the problem here is that what is returned by the backend must be joined together. It returns name and type, and to display them we must change it to type.name. 

I don't see how it would be a good idea to make a component custom for this specific case, when simply returning the correct object is much easier. We could also just add the object "label" to the original element which joins the two other properties

![image](https://user-images.githubusercontent.com/42552874/79599853-359a6180-80b4-11ea-9d2e-59f8de8ab721.png)

It's flexible enough to handle most scenarios, for example here it fetches the languages in a bit more complex object: 

![image](https://user-images.githubusercontent.com/42552874/79599741-084db380-80b4-11ea-9079-2fda2eb63140.png)

With this configuration: 

![image](https://user-images.githubusercontent.com/42552874/79599769-13a0df00-80b4-11ea-9e2a-e427d3be3988.png)